pkg/dataplane/service.go

```diff
@@ -47,15 +47,13 @@ type ServiceYAML struct {
 // DeployService service deployment
 func (d *Deployer) DeployService(foundService dataplanev1.OpenStackDataPlaneService) error {
 	err := dataplaneutil.AnsibleExecution(
-		d.Ctx,
```

Comment:
Why are you removing context across these changes?

Reply:
True, not sure why I did that. Added it back, and changed the AnsibleExecution() code back to using the provided context instead of TODO()